fastexcel-reader/src/test/java/org/dhatim/fastexcel/reader/SimpleReaderTest.java

```diff
@@ -134,4 +134,21 @@ public void testDefaultWorkbookPath2() throws IOException {
             }
         }
     }
+
+    @Test
+    public void testCaseInsensitiveInFileNames() throws IOException {
+        try (InputStream is = Resources.open("/xlsx/caseInsensitive.xlsx");
+             ReadableWorkbook wb = new ReadableWorkbook(is, new ReadingOptions(false, true))) {
+            Sheet sheet = wb.getFirstSheet();
+            try (Stream<Row> rows = sheet.openStream()) {
+                Iterator<Row> it = rows.iterator();
+                assertTrue(it.hasNext());
+                Iterator<Cell> cellIt = it.next().iterator();
+                assertTrue(cellIt.hasNext());
+                Cell cell = cellIt.next();
+                assertEquals(CellType.STRING, cell.getType());
+                assertEquals("A", cell.getValue());
+            }
+        }
+    }
 }
```

Comment:
I cannot see anything case-related in this test. Did I miss something?

Reply:
To understand what need to be case insentive for some xlsx files, please look at inside xlsx file
```
mathieu@localhost:~$ unzip caseInsensitive.xlsx
Archive:  caseInsensitive.xlsx
  inflating: [Content_Types].xml     
   creating: docProps/
  inflating: docProps/core.xml       
  inflating: docProps/app.xml        
   creating: _rels/
  inflating: _rels/.rels             
   creating: xl/
  inflating: xl/workbook.xml         
  inflating: xl/styles.xml           
  inflating: xl/sharedStrings.xml    
   creating: xl/_rels/
  inflating: xl/_rels/workbook.xml.rels  
   creating: xl/worksheets/
  inflating: xl/worksheets/sheet1.xml  

mathieu@localhost:~$ cat [Content_Types].xml
<?xml version="1.0" encoding="UTF-8"?>
<Types xmlns="http://schemas.openxmlformats.org/package/2006/content-types"><Override PartName="/_rels/.rels" ContentType="application/vnd.openxmlformats-package.relationships+xml"/><Override PartName="/docProps/app.xml" ContentType="application/vnd.openxmlformats-officedocument.extended-properties+xml"/><Override PartName="/docProps/core.xml" ContentType="application/vnd.openxmlformats-package.core-properties+xml"/><Override PartName="/xl/_rels/workbook.xml.rels" ContentType="application/vnd.openxmlformats-package.relationships+xml"/><Override PartName="/xl/sharedstrings.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.sharedStrings+xml"/><Override PartName="/xl/worksheets/sheet1.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.worksheet+xml"/><Override PartName="/xl/styles.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.styles+xml"/><Override PartName="/xl/workbook.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet.main+xml"/>
</Types>
```

xl/sharedStrings.xml  file got S, but in [Content_Types].xml we got 
`<Override PartName="/xl/sharedstrings.xml" ContentType="application/vnd.openxmlformats-officedocument.spreadsheetml.sharedStrings+xml"/>`